simtools/corsika/corsika_default_config.py

```diff
@@ -0,0 +1,217 @@
+from collections import defaultdict
+
+import astropy.units as u
+from scipy.interpolate import interp1d
+
+
+class CorsikaDefaultConfig:
+    """
+    This class contains the default configuration for CORSIKA parameters for
+    the various primary particles. It includes all basic dependencies on zenith angles, etc.
+    """
+
+    def __init__(self, primary=None, zenith_angle=None):
+        """
+        Initialize CorsikaDefaultConfig.
+        """
+        self._primary = primary
+        self._zenith_angle = zenith_angle
+        self._energy_slope = -2.0
+        self.energy_ranges = self._define_hardcoded_energy_ranges()
+        self.number_of_showers = self._define_hardcoded_number_of_showers()
+
+    @property
+    def primary(self):
+        """
+        Primary particle.
+        """
+        return self._primary
+
+    @primary.setter
+    def primary(self, primary):
+        """
+        Set primary particle.
+        """
+        supported_primaries = [
```

Comment:
We agree to use strings for particle names and not e.g., PDG names (would not be sure about gamma_diffuse then).

Reply:
There was a "?" missing - we agreed to use those strings and not PDG codes? Fine with me, just want to mention it.

Reply:
Oh, for the configuration we can use whatever we want essentially and we should use these strings as they are easy and standard. I don't think the discussion about PDG usage is relevant for this specific case, but more about the output we pass to the analysis software.

Reply:
OK!